modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/model/handlers.scala

```diff
@@ -503,6 +503,12 @@ object handlers {
       case _                        => false
     }
 
+    val soundCheck: PartialFunction[Any, ActionResult[M]] = {
```

Comment:
It looks so weird to see a `PartialFunction` with domain `Any`. Do you have to do this?
Could it be `Action`?

Reply:
I've tried that but it doesn't seem possible as the `handle` function in `ActionHandler` is defined as:
```
  protected def handle: PartialFunction[Any, ActionResult[M]]
```

and we need to override it